Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -252,6 +254,33 @@ namespace xr
             std::vector<Frame::Plane> Planes{};
             std::vector<FeaturePoint> FeaturePointCloud{};
         } ActionResources{};
+            
+        struct HandJointData {
+            void InitializeXRHandJoints()
+            {
+                locations.jointCount = XR_HAND_JOINT_COUNT_EXT;
+                locations.jointLocations = jointLocations;
+            };
+
+            XrHandJointLocationEXT jointLocations[XR_HAND_JOINT_COUNT_EXT];
+            XrHandJointLocationsEXT locations{XR_TYPE_HAND_JOINT_LOCATIONS_EXT};
+        };
+
+        struct HandInfo {
+            XrHandEXT hand{};
+            XrHandTrackerEXT handTracker{};
+            XrSpace handSpace{};
+
+            HandJointData handJointData{};
+        };
+        
+        struct
+        {
+            XrBool32 supportsHandTracking{ false };
+            XrBool32 HandsInitialized{ false };
+
+            std::array<HandInfo, 2> handsInfo;
+        } HandData;
```

Comment:
nit: formatting consistency, put the struct name at the beginning instead of the end

Reply:
I cleaned it up a little, but this is an anonymous struct used for a single variable, HandData. HandData is a named struct since I don't want to try embedding the struct definition inside the std::array...